admin-manual/installation-setup/customization/instrumentation.rst

```diff
@@ -0,0 +1,337 @@
+* ``django_model_delete_total`` (Counter): Total model save calls labeled by model class
+
+The dashboard also includes additional metrics:
+common_db_retry_time_seconds (Counter): Total time waiting to retry database transactions in seconds, labeled by operation description
+common_ss_api_request_duration_seconds (Counter): Total time waiting on the Storage Service API in seconds, labeled by function
+
+.. _storage-service-metrics:
+
+Storage Service metrics
++++++++++++++++++++++++
+
+The Storage Service includes all the same Django-based metrics as the Dashboard.
+
+.. _instrumentation-configuration:
+
+Configuration
+-------------
+
+The following environment variables will turn on metrics capturing and export in
+Archivematica (provided with example values):
+
+* ``ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROMETHEUS_BIND_PORT=7998``
+* ``ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROMETHEUS_BIND_ADDRESS=0.0.0.0``
+* ``ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_PROMETHEUS_BIND_PORT=7999``
+* ``ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_PROMETHEUS_BIND_ADDRESS=0.0.0.0``
+* ``ARCHIVEMATICA_DASHBOARD_DASHBOARD_PROMETHEUS_ENABLED=true``
+* ``SS_PROMETHEUS_ENABLED=true``
+
+For more documentation on these, see the README files for the `Dashboard`_,
+`MCPClient`_, `MCPServer`_, and `Storage Service`_ (make sure to select the
+correct branch from the branch dropdown menu if you are not using Archivematica
+qa/1.x / Storage Service qa/0.x). Note that you don’t need to export metrics for
+all services.
+
+If you are running multiple MCPClients, each of them must have a different port
+set.
+
+Once the environment variables have been set, restart the Archivematica
+services and confirm that you can access the ``/metrics`` path on each of the
+ports given (e.g. ``http://<am-hostname>:62080/metrics``). It should return a
+block of text that looks like this:
+
+.. code:: bash
+
+   # HELP django_http_requests_latency_seconds_by_view_method Histogram of request processing time labelled by view.
+   # TYPE django_http_requests_latency_seconds_by_view_method histogram
+   django_http_requests_latency_seconds_by_view_method_bucket{le="0.005",method="POST",view="components.api.views.wrapper"} 0.0
+   django_http_requests_latency_seconds_by_view_method_bucket{le="0.01",method="POST",view="components.api.views.wrapper"} 0.0
+
+.. _configure-prometheus:
+
+Configure Prometheus
+++++++++++++++++++++
+
+Once Archivematica is configured to export variables, you can set up Prometheus
+to scrape data from it.
+
+First, `download and install Prometheus`_.
+
+Create a Prometheus configuration file (in this example, it's a YAML file called
+``prometheus.yml``). If you are running all Archivematica components on a single
+host, a good starting point is:
+
+.. code:: yaml
+
+   scrape_configs:
+   - job_name: archivematica
+     scrape_interval: 5s
+     static_configs:
+     - targets:
+	     - localhost:7998
+	     - localhost:7999
+	     - localhost:62080
+	     - localhost:62081
```

Comment:
```suggestion
     - targets:
       - localhost:7998
       - localhost:7999
       - localhost:62080
       - localhost:62081
```

Reply:
Haha thanks @replaceafill, I actually made a mental note of that and then completely forgot to fix it! Was super tired last night.